github/commitstatus.go

```diff
@@ -107,6 +109,7 @@ func (s CommitStatus) Add(sha, state, targetURL, description string) error {
 	req.Header.Set("Authorization", "token "+s.token)
 	req.Header.Set("Accept", "application/vnd.github.v3+json")
 	req.Header.Set("Content-Type", "application/json")
+	req.Header.Set("X-GitHub-Api-Version", "2022-11-28")
```

Comment:
per the review comments, this is not needed.

Reply:
Removed as suggested.